openshift/template.yaml

```diff
@@ -0,0 +1,251 @@
+apiVersion: v1
+kind: Template
```

Comment:
Is this an Openshift API resource? :thinking:  never heard of template on Kubernetes. If yes, we might have to modify my saas file PR on gitlab, because I didn't specify that API resource there yet.

Reply:
nope, no need. this is the file they process.